care/facility/api/viewsets/facility.py

```diff
@@ -43,8 +39,8 @@ def filter(self, qs, value):
 
 class FacilityFilter(filters.FilterSet):
     name = filters.CharFilter(field_name="name", lookup_expr="icontains")
-    facility_type = filters.NumberFilter(field_name="facility_type")
-    organization = GeoOrganizationFilter()
+    facility_type = filters.BaseInFilter(field_name="facility_type", lookup_expr="in")
```

Comment:
Why was this changed here?

Reply:
This was updated to support filtering by multiple facility types, exact use case was to filter labs, both private and public
<img width="560" alt="image" src="https://github.com/user-attachments/assets/8d1a8852-465d-4c39-bed1-91ab1269dd70" />